opencti-platform/opencti-graphql/src/database/engine.js

```diff
@@ -683,12 +683,13 @@ const updateIndexTemplate = async (name, mapping_properties) => {
 };
 
 const elCreateIndexTemplate = async (index, mappingProperties) => {
-  const isExistByName = await engine.indices.existsIndexTemplate({ name: index }).then((r) => oebp(r));
   // Compat with platform initiated prior 5.9.X
-  const isExist = await engine.indices.existsIndexTemplate({ name: `${ES_INDEX_PREFIX}-index-template` }).then((r) => oebp(r));
-  if (isExistByName || isExist) {
+  const isPriorVersionExist = await engine.indices.existsIndexTemplate({ name: `${ES_INDEX_PREFIX}-index-template` })
```

Comment:
I don't understand this condition, shouldn't it be `{ name: index }` ?

Reply:
Its a condition i have kept for older platform versions that doesnt have the new templates.